api/v1alpha1/generated.proto

```diff
@@ -862,6 +862,16 @@ message RouteBindingSelector {
   //
   // +optional
   optional k8s.io.apimachinery.pkg.apis.meta.v1.LabelSelector routeSelector = 2;
+
+  // Kind specifies the kind of resource to select with the above selectors.
+  //
+  // Kind MUST correspond to route resources that are compatible with the
```

Comment:
Do we want to default group and version to the service-apis? We do have a lot of other references with defaults so there is some consistency there and it makes things a bit simpler for the "standard" httproute case.

I don't have a strong opinion here but seems like it might be a bit better ux

Reply:
I do like the idea of defaulting but maybe we could make it protocol dependent and default all of GVK, instead of just the GV? E.g. for HTTP(S) default to HTTPRoute.